winlogbeat/beater/eventlogger.go

```diff
@@ -144,7 +145,8 @@ func (e *eventLogger) run(
 		client:     client,
 		eventACKer: eventACKer,
 	}
-	if err := eventlog.Run(noopReporter{}, ctx, api, state, publisher, e.log); err != nil {
+	reg, _ := inputmon.NewInputRegistry("winlog", api.Name(), nil)
```

Comment:
We still use the old `inputmon.NewInputRegistry` here since this is the path for standalone winlogbeat, and then pass it through to `eventlog.Run`